### PR TITLE
Mario/plutarch 1.2 cleanups

### PR DESCRIPTION
**Pull request by blamario**

### CODE REVIEW

File: Plutarch/Bool.hs [hunk 1]
```diff
@@ -155,22 +153,17 @@ gpeq =
...
-
-gpeq' ::
-  forall a s.
-  All2 PEq (PCode s a) =>
-  SOP (Term s) (PCode s a) ->
-  SOP (Term s) (PCode s a) ->
-  Term s PBool
+            gpeq' (gpfrom x') (gpfrom y')
+
+gpeq' :: All2 PEq a => SOP (Term s) a -> SOP (Term s) a -> Term s PBool
```
Comment (srid) [inline Plutarch/Bool.hs:158]: For clarity, I'd `s/a/xss`.

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +14,12 @@
 
   Started by [#335](https://github.com/Plutonomicon/plutarch/pull/335)
 
+- Added module `Plutarch.FFI` for interoperability with PlutusTx.
+
+- Added module `Plutarch.Show` with the `PShow` type class.
```
Comment (srid) [inline CHANGELOG.md:19]: A similar entry (along with the link to the PR) was already added to the top of the CHANGELOG. You can merge them.

File: Plutarch/Bool.hs (hunk 1 above)
Comment (blamario) [reply Plutarch/Bool.hs:158]: done

**Review by L-as — APPROVED** [review]
LGTM

**Comment by L-as** [comment]
There's a conflict

**Comment by blamario** [comment]
If I had merged instead of rebasing, would the approval have stayed?

**Comment by L-as** [comment]
If the commit changes the approval will disappear regardless of how you did it AFAIK.

**Review by L-as — APPROVED** [review]
LGTM

**Pull request merged**